linera-chain/src/block.rs

```diff
@@ -109,3 +112,21 @@ impl From<Timeout> for HashedCertificateValue {
         HashedCertificateValue::new_timeout(value.chain_id, value.height, value.epoch)
     }
 }
+
+impl From<HashedCertificateValue> for Hashed<Timeout> {
+    fn from(hv: HashedCertificateValue) -> Hashed<Timeout> {
```

Comment:
nit: (See naming guidelines) We don't use acronyms so it should be `value: HashedCertificateValue` or `(hashed_)certificate_value: HashedCertificateValue`.

Reply:
Done in [Replace From with TryFrom](https://github.com/linera-io/linera-protocol/pull/2887/commits/bdbad2f527b1de1ac3a773dee42ccd6d40577289)